.ci/docker/common/install_inductor_benchmark_deps.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -ex
+
+source "$(dirname "${BASH_SOURCE[0]}")/common_utils.sh"
+
+function install_huggingface() {
+  local version
+  version=$(get_pinned_commit huggingface)
+  pip_install pandas
+  pip_install scipy
+  pip_install "transformers==${version}"
+}
+
+function install_timm() {
+  local commit
+  commit=$(get_pinned_commit timm)
+  pip_install pandas
+  pip_install scipy
+  pip_install "git+https://github.com/rwightman/pytorch-image-models@${commit}"
```

Comment:
From the error https://hud.pytorch.org/pr/pytorch/pytorch/102881#13970022132, this step pulls in `cmake=3.26.3` which is not compatible with CI atm.  You would need to do something similar like this https://github.com/pytorch/pytorch/blob/main/.ci/docker/common/install_triton.sh#L50-L62 to restore cmake version after the installation